### ISSUE TITLE
Hog's Back Site #3 Workback Plan

### ISSUE THREAD

**Issue by MartinNesbitt** [ForetCapitaleForest/actions#569]
Planting Opportunity at a 3rd site; CIMA + partnership opportunity – Hog’s Back is to be confirmed.
NCC confirmed the strip at Hog’s Back, portion that extends all the way to Hog’s Back Road. Estimate about 300 trees in that area according to Cedric. Note that middle part is a very wet zone/ marshy. Cedric will revisit by making a request with the land manager.

### Planning
| Date | Activity &#9744; &#9745;| Owner | Issue # |
| --- | --- | --- | --- |
|  | &#9744; Receive Tree Delivery | @Matthewtennant |
| three weeks out  | &#9744;  Send out first CIMA+ email | @ShellCrabtree @MartinNesbitt   |
| three weeks out  | &#9744;  Pick tree order to acclimatize | @Matthewtennant  |
| one week out   | &#9744;  Confirm mulch delivery to site |   |
| one week out  | &#9744;  Reach out to Nick and Raymond about tree delivery | @KaviGK @MartinNesbitt  |
| one week out  | &#9744; Week Before email to CIMA+| @ShellCrabtree @MartinNesbitt   |  |
| one day before | &#9744; Reminder email to CIMA+ | @ShellCrabtree @MartinNesbitt   | |

### Day of Event - 
 | Activity &#9744; &#9745;| Owner | Issue # |
 | --- | --- | --- |
 |  &#9744; last-chance weather cancellation |  |  |
 | &#9744; Tree Transport | @Matthewtennant @MartinNesbitt  |  |
 | &#9744; Hogs Back Site 3 Event  | @MartinNesbitt @KaviGK  | |
| &#9744; Post to Social Media | @KaviGK  |  |

### Post-Event
 | Date | Activity &#9744; &#9745;| Owner | Issue # |
| --- | --- | --- | --- |
| day after | &#9744; Event post-mortem | @MartinNesbitt @ShellCrabtree   |  |
| week after | &#9744; thank you email to CIMA+ | @ShellCrabtree @MartinNesbitt   |  |
| month after | &#9744; tree after-care campaign | @MartinNesbitt @KaviGK  |  |
|  | &#9744; Site photography |  @MartinNesbitt @KaviGK  |  |
|  | &#9744; Monitoring and Maintenance |  @MartinNesbitt  |  |

**Comment by ShellCrabtree** [comment]
C. Bertrand - confirm approval by email:
I'd like to take this opportunity to remind you of the site approval we've obtained for Hog's Back. You could organize a planting of 200 to 300 trees there as early as this spring, or rather in the autumn if you prefer.  We can discuss the details when you get a chance.

**Comment by jodinewman** [comment]
Tree species list approved, waiting to confirm funding with CIMA+ to order trees. Species list is as follows:

| Species | Quantity | Source | 
| -------- | ---------- | -------- | 
| _Acer rubrum_ (Red maple) | 50 | Baker's (need to order) |
| _Quercus rubra_ (Northern red oak) | 50 | Baker's (in seedling area now) | 
| _Quercus alba_ (White oak) | 30 | Baker's (in seedling area now) | 
| _Tilia americana_ (Basswood) | 50 | Need to find source |
| _Ostrya virginiana_ (Ironwood) | 40 | Need to find source |
| _Betula alleghaniensis_ (Yellow birch ) | 40 | NVK (need to order) |
| _Populus grandidentata_ (Largetooth aspen) | 40 | Need to find source |

**Comment by Matthewtennant** [comment]
waiting on confirmation of event before creating individual tasks

**Comment by Matthewtennant** [comment]
event postponed until the fall
